### ISSUE TITLE
Radio button accessibility issue

### ISSUE THREAD

**Issue by Bilal3120** [SAP/ui5-webcomponents#5452]
**Describe the bug**
The accessibleNameRef of radio button not announced

**Isolated Example**
[CodesandBox](https://codesandbox.io/s/radiobuttonlist-zwme90?file=/src/App.js)

**To Reproduce**
1. Go to [CodesandBox](https://codesandbox.io/s/radiobuttonlist-zwme90?file=/src/App.js)
2. Click on Button1 or Button2
3. The screen reader does not announce the accessibleNameRef component which is a Label

**Expected behavior**
Label is announced along with text of the radio button

**Screenshots**
![MicrosoftTeams-image (7)](https://user-images.githubusercontent.com/93638146/177112497-2f165ca7-ddf0-4ec2-868e-c879084fe2fb.png)

**UI5 Web Components for React Information**
`@ui5/webcomponents` 1.4.0:
`@ui5/webcomponents-react` 0.25.7:
Operating System: Mac OS
Browser: Chrome
